### ISSUE TITLE
Viewinspector seems broken after updating Xcode to 14.3 / iOS 16.4

### ISSUE THREAD

**Issue by ztolley** [nalexn/ViewInspector#235]
I have an existing project that has stopped working with view inspector after updating Xcode and when I try to start a new project it won't build when I add view inspector:

## New project - Steps to reproduce
* Upgrade to Xcode 14.3 with it's support for iOS 16.4
* Create a new iOS swift application with tests
* Build and run application, works fine
* Run unit test, works fine
* Go to project, package dependencies and add ViewInspector from GitHub (0.9.5)
* Rebuild project, fails.

<img width="1605" alt="Screenshot 2023-04-01 at 13 02 21" src="https://user-images.githubusercontent.com/56056/229287451-3e43ad0a-7a8e-4115-a033-1892125bc889.png">

## Existing project  (set to iOS )
* Create a simple layout with a text field that uses accessibility labels and identifiers
* Create tests to find and test the text element using search search and queries to find by identifiers. Searching by text works but identifier based tests fail. 

<img width="1633" alt="Screenshot 2023-04-01 at 13 15 20" src="https://user-images.githubusercontent.com/56056/229288227-560faf3d-fc5a-46de-88dd-a4c187119dbe.png">

Before updating to Xcode 14.3 tests to find things by the identifiers worked, I had a whole suite of tests that used that method to test the content of a thing, e.g.

```{swift}
func testTemp() throws {
    let view = TackDetail(sailDetail: getSailDetail())

    XCTAssertEqual(try
                   view.inspect()
        .find(viewWithAccessibilityIdentifier: "timeAboveMaxTemp")
        .text()
        .string()
    , "17460")
}
```

**Comment by kpacholak** [comment]
Hi. I also had a problem after upgrading to Xcode 14.3
```Toggle's tap() and isOn() are currently unavailable for inspection on iOS 16. Situation may change with a minor OS version update. In the meanwhile, please add XCTSkip for iOS 16 and use an earlier OS version for testing.```

<img width="801" alt="Zrzut ekranu 2023-04-1 o 18 05 11" src="https://user-images.githubusercontent.com/63740434/229301755-4d12ab44-6038-47c2-a880-30605b4f6f31.png">

**Comment by Tyler-Keith-Thompson** [comment]
I think your XCTest symbol issue is a red herring. I just tried to repro this and didn't see the issue. It's possible you accidentally added ViewInspector to your app target instead of your unit testing target. I'm trying to debug the `find` issues, though.

**Comment by ztolley** [comment]
I'll recreate a new project but you could be right. The find issue is definitely a thing though.

**Comment by Tyler-Keith-Thompson** [comment]
Once #237 is merged it should be fixed :)

**Comment by nalexn** [comment]
@ztolley I've merged #237. I want to get a couple more fixes in before publishing the new version, in the meanwhile, could you verify the fix works for you?

**Comment by ztolley** [comment]
I'll give it a try later this morning , thank you

**Comment by nalexn** [comment]
And yeah, the XCTest symbol issue usually appears when the package is linked with the wrong target (should be test target only).

**Comment by ztolley** [comment]
Tested and works

**Comment by svattap** [comment]
@nalexn Do you have tentative date in mind for the new release?

**Comment by mrk-han** [comment]
+1 to this, just commenting so I can offer to help with any testing or part of the release process if needed @nalexn

**Comment by nalexn** [comment]
Sorry guys for holding it, I'll do it shortly this week

**Comment by nalexn** [comment]
Published with 0.9.6 release
